nautobot/users/migrations/0011_create_default_groups_and_permissions.py

```diff
@@ -0,0 +1,70 @@
+from django.contrib.auth.models import Group
```

Comment:
I think this is the wrong branch to push up these changes. I think you meant to do it for this [PR](https://github.com/nautobot/nautobot/pull/6766) and we decided to close it because https://github.com/nautobot/nautobot/pull/6766#issuecomment-2593394276